### PR TITLE
100K user Galaxy EU

### PR DESCRIPTION
**Pull request by erxleben**

### CODE REVIEW

File: content/news/2024-07-29-100K-Users-GalaxyEU/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: '100,000 users for Galaxy-EU'
+date: '2024-07-29'
+tease: "We are thrilled to announce, that UseGalaxy.EU has reached 100,000 users!"
+hide_tease: false
+authors: Anika Erxleben-Eggenhofer 
+subsites: [all-eu, global]
```
Comment (bgruening) [inline content/news/2024-07-29-100K-Users-GalaxyEU/index.md:7]:
```suggestion
subsites: [all-eu, global]
tags: [esg]
```

**Comment by wm75** [comment]
should we keep the number of histories consistent now between hereand @erxleben's most recent version?

**Comment by bgruening** [comment]
Sure, do you have a better image?

**Review by bgruening — APPROVED** [review]
Feel free to merge!

**Comment by bgruening** [comment]
Oh I found automerge :)

**Pull request merged**